src/views/Accessibility/AccessibilityRequestDetailPage.tsx

```diff
@@ -321,6 +338,35 @@ const AccessibilityRequestDetailPage = () => {
           }}
         </Formik>
       </div>
+      <div
+        role="region"
+        aria-label="existing notes"
+        className="margin-top-6 margin-x-1"
+      >
+        <div
+          className="usa-sr-only"
+          id="AccessibilityRequestNotesTab-NotesList"
+        >
+          existing notes
+        </div>
+        <NotesList>
+          {data?.accessibilityRequest?.notes?.map(
+            note =>
+              note && (
```

Comment:
Is `note &&` necessary? Is it possible for GraphQL to return a falsy value?

Reply:
I added a workaround.